NOTICE

```diff
@@ -1,667 +1,11 @@
 Apache Spark
-Copyright 2014 and onwards The Apache Software Foundation.
+Copyright 2014 - 20018 The Apache Software Foundation.
```

Comment:
2018?

Reply:
Yes 2018 as that when the last release was. Copyright expires so it nice to know what the year was.

Reply:
Ah I see what you are saying - I'll fix it to be 2018 not 20018